types/apollo-codegen/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for apollo-codegen 0.16
+// Project: https://github.com/apollographql/apollo-codegen
+// Definitions by: Bradley Ayers <https://github.com/bradleyayers>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
It can be Typescript 2.5 here

Reply:
I thought it would be desirable to have the oldest compatible version?